src/pages/clients/create/Create.tsx

```diff
@@ -66,9 +66,23 @@ export function Create() {
 
   const onSave = () => {
     set(client as Client, 'contacts', contacts);
-
     const toastId = toast.loading(t('processing'));
+    setErrors(undefined);
+    if (
+      !(
+        client?.name != '' ||
+        contacts[0].first_name != '' ||
+        contacts[0].last_name != ''
```

Comment:
Requirements have not changed since the original task. In the original commit, only the client name was referenced for this, so I am not entirely sure where the validation for contact came from. Even more, why only validate the first contact. 

Please stick to original requirements and validate the client name only.